Assets/MixedRealityToolkit.SDK/Inspectors/Input/Handlers/SpeechInputHandlerInspector.cs

```diff
@@ -20,13 +20,15 @@ public class SpeechInputHandlerInspector : BaseInputHandlerInspector
 
         private SerializedProperty keywordsProperty;
         private SerializedProperty persistentKeywordsProperty;
+        private SerializedProperty speechConfirmationTooltipPrefabProperty;
 
         protected override void OnEnable()
         {
             base.OnEnable();
 
             keywordsProperty = serializedObject.FindProperty("keywords");
             persistentKeywordsProperty = serializedObject.FindProperty("persistentKeywords");
+            speechConfirmationTooltipPrefabProperty = serializedObject.FindProperty("speechConfirmationTooltipPrefab");
```

Comment:
Can we instead have the following:

1) a checkbox "use speech confirmation tooltip"
2) If checked, the speech confirmation tooltip prefab field shows up.

Or, have a tooltip that clearly states "if null, no tooltip will show up".

Reply:
The tooltip text is in SpeechInputHandler.cs. 
 
       [SerializeField]
        [Tooltip("Assign SpeechConfirmationTooltip.prefab here to display confirmation label. Optional.")]
        private GameObject speechConfirmationTooltipPrefab = null;